src/graphql/schema/Playlist/playlist.js

```diff
@@ -1,18 +1,56 @@
 import { gql } from 'apollo-server-express';
+import Playlist from '../../../models/Playlist';
 
 // Type definitions for Playlists go here
 export const typeDefs = gql`
   extend type Query {
-    playlists: [Playlist!]!
+    getPlaylists: [Playlist]!
   }
+
+  extend type Mutation {
+    createPlaylist(name: String!, description: String): Playlist
+    addTracks(playlistId: String, tracks: [String]): SnapshotID
+  }
+
   type Playlist {
-    playlist: String!
+    name: String!
+    description: String
```

Comment:
Should there be an exclamation point here?

Reply:
For description? No, because the description is optional. The only thing required according to the Spotify API is name.